src/dataset-builder/TestConstants.ts

```diff
@@ -10,38 +10,27 @@ export const dummyDatasetDetails = (datasetId: string): DatasetModel => ({
   properties: {},
   snapshotBuilderSettings: {
     programDataOptions: [
-      { id: 1, name: 'Year of birth', kind: 'range', min: 1900, max: 2023 },
+      { id: 1, name: 'Year of birth', kind: 'range', tableName: 'person', columnName: 'year_of_birth' },
```

Comment:
it seems like this isn't all dummy data or testConstants anymore. Is that right? I'm wondering as this process continues and stubbed data is replaced, is renaming needed to match the end goal?

Reply:
This data is still only used by test code. Even when we have stubbed APIs, there is an API that returns this object so we'll need the larger dummy object for tests of that API.